code/game/objects/structures/safe.dm

```diff
@@ -2,8 +2,12 @@
 CONTAINS:
 SAFES
 FLOOR SAFES
+Safe Codes
+Safe Internals
 */
 
+var/list/global_safes = list()

```

Comment:
`GLOBAL_LIST_EMPTY()`

Reply:
Could you give a little more context on this one, I am unsure the correction desired.